logging/logging.go

```diff
@@ -700,7 +700,8 @@ func fromHTTPRequest(r *HTTPRequest) *logtypepb.HttpRequest {
 		return nil
 	}
 	if r.Request == nil {
-		panic("HTTPRequest must have a non-nil Request")
+		log.Println("Error: HTTPRequest must have a non-nil Request")
```

Comment:
I think just returning nil in this case is valid, no need to log. Also, could we add a test to show this behavior is intentional.

Reply:
Sgtm! I added a test case for it.

If it's ok, I'll keep the `stderr` log for now since this is an easy user error to make, and having a log to `stderr` is helpful.

Reply:
SG, the reason I brought it up is we don't really have much logging in any of our clients. And what little we do we don't have hooks for our users to turn on/off or adjust verbosity.

Reply:
Noted as a future improvement to let users adjust log verbosity ~